focus-group.js

```diff
@@ -37,12 +40,16 @@ function getToolbarItems(group) {
       item.role === 'button' ||
       item.type === 'button' ||
       item.role === 'checkbox' ||
-      item.type === 'checkbox'
+      item.type === 'checkbox' ||
+      item.hasAttribute('tabindex')
```

Comment:
Let's just ignore `tabindex` since we anyway ignore it in the order

Reply:
If I remove the `tabindex` here, we won't be able to navigate through non-focusable elements (like div or span). Is that ok?

Reply:
Yes. We already added this limitation for other case, so let's keep it since we need a lot of code to support this edge case really (like track order of `tabindex`)

Reply:
@ai fixed